cli/cmd/helm/upgrade.go

```diff
@@ -182,7 +182,8 @@ func upgradeTobs(cmd *cobra.Command, args []string) error {
 	out := exec.Command("helm", cmds...)
 	k, err := out.CombinedOutput()
 	if err != nil {
-		return fmt.Errorf("failed to upgrade %v", string(k))
+		fmt.Println(string(k))
```

Comment:
should this be removed?

Reply:
I wanted this. As we need more info on failures. i.e. With `%v` or `%w` all we get is `exit status 1`. By adding the additional string output `gives us the exact output emitted by `helm`. 

I will change this to `return fmt.Errorf("failed to upgrade %s: %w", output, err)` for better logging on error.